test/util/upgrade.js

```diff
@@ -465,17 +483,13 @@ async function populateProtocolContract(
 
       // Default offer is in native token. Change every other to mock token
       if (offerId % 2 == 0) {
-        offer.exchangeToken = mockToken.address;
+        offer.exchangeToken = await mockToken.getAddress();
       }
 
       // Set unique offer dates based on offer id
       const now = offerDates.validFrom;
-      offerDates.validFrom = ethers.BigNumber.from(now)
-        .add(oneMonth + offerId * 1000)
-        .toString();
-      offerDates.validUntil = ethers.BigNumber.from(now)
-        .add(oneMonth * 6 * (offerId + 1))
-        .toString();
+      offerDates.validFrom = BigInt(now) + oneMonth + offerId * (1000).toString();
+      offerDates.validUntil = BigInt(now) + oneMonth * 6 * (offerId + 1).toString();
```

Comment:
This concatenates strings, does not add up values.

Reply:
Line 491 still has the same problem.

Reply:
fixed